src/main/resources/assets/locks/lang/zh_cn.json

```diff
@@ -19,7 +19,7 @@
 	"locks.subtitle.pin.match": "栓合适点击声",
```

Comment:
Should I also replace these two Chinese strings with the new English ones? Or maybe it's not worth it?